x/staging/posts/keeper/keeper_reports_test.go

```diff
@@ -4,6 +4,58 @@ import (
 	"github.com/desmos-labs/desmos/x/staging/posts/types"
 )
 
+func (suite *KeeperTestSuite) TestKeeper_CheckReportValidity() {
+	tests := []struct {
+		name      string
+		report    types.Report
+		shouldErr bool
+	}{
+		{
```

Comment:
Can we add a test to make sure that if no reasons are specified, an error is returned as well?